package.json

```diff
@@ -79,13 +79,13 @@
     "vets-json-schema": "git+https://github.com/department-of-veterans-affairs/vets-json-schema.git"
   },
   "peerDependencies": {
-    "react": "^15.5.4",
-    "react-dom": "^15.6.2",
-    "uswds": "^1.6.3"
+    "react": "^15.5.4||^16.7.0",
+    "react-dom": "^15.6.2||^16.7.0",
+    "uswds": "^1.4.2"
```

Comment:
We've been on 1.4 on VA.gov without any issues, so I threw this change in here as well, because we've been seeing that warning for a while.

Reply:
Which warning is this? 1.6.9 fixed a vuln and that version range should use it.

Reply:
Sorry, an incorrect peer dependency warning in our repo. Do you what vulnerability 1.6.9 fixed over 1.4? Our security check isn't flagging any medium severity or higher issues.

Reply:
https://github.com/uswds/uswds/releases/tag/v1.6.9

Is the problem that you're stuck on 1.4.2 and can't go above that for some reason? If so we can drop this down, but it would be better to use the latest if possible.

Reply:
We're currently trying to figure out how we should use USWDS in relation to the VA design system, and the last time someone checked there were some non-trivial changes we had to make to update from 1.4. So we're holding off for the moment. The warning is not a significant thing on our side, I just updated it because I was already changing peer deps. I'm fine changing it back if you all prefer it.

Reply:
USFS doesn't really have any dependency on USWDS that we test, we just put in the names it wants so dropping back is fine with me. You're pulling in the `.scss` files anyway rather than our built `.css` files, when I looked at the problem a few weeks ago it seemed like most library users would need to do that.